templates/layout.html

```diff
@@ -210,7 +118,7 @@
   </footer>
 </template>
 
-<template name="_loginButtonsAdditionalLoggedInDropdownActions">
+<template name="_loginButtonsAdditionalLoggedInDropdownActions"> //TODO This is unused.
```

Comment:
Yeah, I'm not sure what a better API would be for the bootstrap-3 package. Maybe if there wasn't a default template name, and instead you just had to tell it what the template name is?